Bump QD to level 3 and updated QD (#14)
ros2/spdlog_vendor: SPDLOG_QUALITY_DECLARATION.md

### API Stability Policy [1.iv]

> There is no policy for API stability. This is not a problem because the `spdlog_vendor` package importing the `spdlog dependency` is using a fixed version, in this case, the [1.5.0](https://github.com/gabime/spdlog/releases/tag/v1.5.0).

Reviewer: I think there is no problem with pinning a version, but IMO the wording should reflect that it won't affect ROS 2 users of `spdlog_vendor`. @brawner may suggest a better wording.

Author: I used the same wording that you can find in [libyaml](https://github.com/ros2/libyaml_vendor/blob/master/libyaml_q_declaration.md#api-stability-policy-1iv)

Reviewer 2: ```suggestion
There is no policy for API stability. However, the vendored package `spdlog_vendor` package importing the `spdlog dependency` is using a fixed version, in this case, [1.5.0](https://github.com/gabime/spdlog/releases/tag/v1.5.0), which will provide API stability to downstream packages.
```

> There is no policy for API stability. However, the vendored package `spdlog_vendor` package importing the `spdlog dependency` is using a fixed version, in this case, the [1.5.0](https://github.com/gabime/spdlog/releases/tag/v1.5.0), which will provide API stability to downstream packages.

Reviewer 2: ```suggestion
There is no policy for API stability. However, the vendored package `spdlog_vendor` package importing the `spdlog dependency` is using a fixed version, in this case [1.5.0](https://github.com/gabime/spdlog/releases/tag/v1.5.0), which will provide API stability to downstream packages.
```

Also, does it even make sense to mention the specific version here. Perhaps instead you should link to where the current version can be found?

### ABI Stability Policy [1.v]

> There is no policy for ABI stability. This is not a problem because the `spdlog_vendor` package importing the `spdlog` dependency is using a fixed version, in this case, the [1.5.0](https://github.com/gabime/spdlog/releases/tag/v1.5.0).

Reviewer 2: This should also be adjusted, I think

> There is no policy for ABI stability. This is not a problem because the `spdlog_vendor` package importing the `spdlog` dependency is using a fixed version, in this case, the [1.5.0](https://github.com/gabime/spdlog/releases/tag/v1.5.0).

Reviewer 2: ```suggestion
There is no policy for ABI stability. This is not a problem because the `spdlog_vendor` package importing the `spdlog` dependency is using a fixed version, in this case [1.5.0](https://github.com/gabime/spdlog/releases/tag/v1.5.0).
```

### ABI and ABI Stability Within a Released ROS Distribution [1.vi]

> There is not a direct correlation between the `spdlog` releases and the ROS distributions, however this is not a problem because the `spdlog_vendor` package importing the `spdlog` dependency is using a fixed version, in this case, the [1.5.0](https://github.com/gabime/spdlog/releases/tag/v1.5.0).

Reviewer 2: ```suggestion
`spdlog`'s API and ABI stability within a ROS Distribution are maintained by `spdlog_vendor` which pins the specific version of the `spdlog` dependency to a specific version, in this case, [1.5.0](https://github.com/gabime/spdlog/releases/tag/v1.5.0).
```

> `spdlog`'s API and ABI stability within a ROS Distribution are maintained by `spdlog_vendor` which pins the specific version of the `spdlog` dependency to a specific version, in this case, [1.5.0](https://github.com/gabime/spdlog/releases/tag/v1.5.0).

Reviewer 2: ```suggestion
`spdlog`'s API and ABI stability within a ROS Distribution are maintained by `spdlog_vendor` which pins the `spdlog` dependency to a specific version, in this case [1.5.0](https://github.com/gabime/spdlog/releases/tag/v1.5.0).
```

Sorry, my suggestion last week wasn't the best.
